sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java

```diff
@@ -641,16 +641,16 @@ PollerFlux<DocumentOperationResult, DocumentModelDetails> beginComposeModel(List
      * @throws NullPointerException If {@code modelId} or {@code target} is null.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<DocumentOperationResult, DocumentModelDetails> beginCopyModelTo(String modelId,
+    public PollerFlux<DocumentOperationResult, DocumentModelDetails> beginCopyModelTo(String sourceModelId,
```

Comment:
Was this rename feedback from Java architects? I like it. I think it makes the whole copy story a bit clearer, so I'll probably do the same in .NET.

In the `CopyAuthorizationOptions`, do you also call the model ID `targetModelId`?

Reply:
>In the CopyAuthorizationOptions, do you also call the model ID targetModelId?

I think that is how it is in the swagger.